TrackingTools/TrajectoryState/src/classes_def.xml

```diff
@@ -1,5 +1,6 @@
 <lcgdict>
   <!-- persistent="false" qualifier for non-wrappers is deprecated. Will be removed when no longer needed.-->
+  <class name="TrajectoryStateOnSurface"/>
```

Comment:
I was not quite sure what persistent does, and the comment just above said it is deprecated. Do you think I should still add it?